### PR TITLE
Release 0.7.1

### PR DESCRIPTION
**Pull request by jeremywiebe**
Update to point to Astro version that removes dependency on `grunt-harp`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "astro-velo",
-  "version": "0.7.0",
+  "version": "0.7.1",
   "author": "Mobify",
   "dependencies": {
     "astro-sdk": "^0.7.0",
```
Comment (kschmidtdev) [inline package.json]: Should it be pointing to 0.7.1 as well?

Comment (jeremywiebe) [inline package.json]: Don't need to anymore.  The `^` means that npm will use the most up-to-date `0.7.*` release of Astro.

Comment (kschmidtdev) [inline package.json]: TIL!

**Comment by kschmidtdev** [comment]
:+1:

**Pull request merged**